framework/arcane-framework/src/main/scala/services/mssql/QueryProvider.scala

```diff
@@ -0,0 +1,195 @@
+package com.sneaksanddata.arcane.framework
+package services.mssql
+
+import java.time.format.DateTimeFormatter
+import java.time.{Duration, Instant, LocalDateTime, ZoneOffset}
+import scala.concurrent.Future
+import scala.io.Source
+import scala.util.{Try, Using}
+
+object QueryProvider:
+  /**
+   * The key used to merge rows in the output table.
+   */
+  private val UPSERT_MERGE_KEY = "ARCANE_MERGE_KEY"
```

Comment:
let's instead use the value from `MergeKeyField`

Reply:
Fixed